builtin/providers/aws/resource_aws_elasticache_cluster_test.go

```diff
@@ -23,6 +23,8 @@ func TestAccAWSElasticacheCluster(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckAWSElasticacheSecurityGroupExists("aws_elasticache_security_group.bar"),
 					testAccCheckAWSElasticacheClusterExists("aws_elasticache_cluster.bar"),
+					resource.TestCheckResourceAttr(
+						"aws_elasticache_cluster.bar", "cache_nodes.0.id", "001"),
```

Comment:
It does now, because of [this line](https://github.com/hashicorp/terraform/pull/1965/files#diff-f37f3a7838a505a835dc5dbf18637927R182) and [this line](https://github.com/hashicorp/terraform/commit/d81e63cc3c2411077ee629d3ca30cebe36ef4464#diff-b1bb3d642d5f167c69e0e18779e559aaR27)